docs/jwst/associations/level2_asn_technical.rst

```diff
@@ -6,20 +6,18 @@ Stage 2 Associations: Technical Specifications
 Logical Structure
 -----------------
 
-Independent of the actual format, all stage 2 associations have the
-following structure. Again, the structure is defined and enforced by
-the stage 2 schema
+All stage 2 associations have the following structure. The structure is defined
+and enforced by the stage 2 schema
```

Comment:
minor nit: missing a period at the end of `schema`

Reply:
✅